store/filter.go

```diff
@@ -701,7 +701,11 @@ type ByFilesFilter interface {
 // or contain any of the listed files. It panics if any file path is
 // empty, or if the file path has not been cleaned (i.e., if file !=
 // path.Clean(file)).
-func ByFiles(files ...string) interface {
+//
+// If specific == true, then only the specific files are accepted (i.e. a
+// directory path will not include all files in that directory as it would
+// normally).
+func ByFiles(specific bool, files ...string) interface {
```

Comment:
LGTM.

On Mon, Jan 4, 2016 at 9:10 PM, Stephen Gutekanst <notifications@github.com>
wrote:

> In store/filter.go
> <https://github.com/sourcegraph/srclib/pull/236#discussion_r48812926>:
>
> > @@ -701,7 +701,11 @@ type ByFilesFilter interface {
> >  // or contain any of the listed files. It panics if any file path is
> >  // empty, or if the file path has not been cleaned (i.e., if file !=
> >  // path.Clean(file)).
> > -func ByFiles(files ...string) interface {
> > +//
> > +// If specific == true, then only the specific files are accepted (i.e. a
> > +// directory path will not include all files in that directory as it would
> > +// normally).
> > +func ByFiles(specific bool, files ...string) interface {
>
> Good idea -- I had trouble coming up with the correct name for this
> parameter. I do like exact much more.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/sourcegraph/srclib/pull/236/files#r48812926>.
>